quickwit-proto/proto/search_api.proto

```diff
@@ -127,8 +135,16 @@ message PartialHit {
 message LeafSearchResult {
   // Total number of documents matched by the query.
   uint64 num_hits = 1;
+
   // List of the best top-K candidates for the given leaf query.
   repeated PartialHit partial_hits = 2;
+
+  // The list of requests that failed. LeafSearchResult can be an aggregation of results, so there may be multiple.
+  repeated SplitSearchError failed_requests = 3;
+
+  // Total number of aggregated results into this result. 
+  uint64 aggregated_results = 4;
```

Comment:
the name is really not helping here. result = hit? aggregate = statistics aggregation result?

Reply:
Better name and comment:

// Total number of splits this `LeafSearchResult` covers.
// num_attempted_splits = num_successful_splits + num_failed_splits.
uint64 num_attempted_splits = 4;

Reply:
yes that's better

Reply:
although leaf may not be completely accurate, since results of multiple leafs are merged at some point